packages/types-algosdk/types/index.d.ts

```diff
@@ -375,7 +375,7 @@ declare module 'algosdk' {
 
   export function makeAssetConfigTxnWithSuggestedParams(from: any, note: any, assetIndex: any,
     manager: any, reserve: any, freeze: any, clawback: any, suggestedParams: any,
-    strictEmptyAddressChecking: any): any;
+    strictEmptyAddressChecking?: any): any;
```

Comment:
Please update this in the above function type as well (`makeAssetConfigTxn`) as well

Reply:
?

Reply:
ah, okay